ruby/Gemfile

```diff
@@ -2,5 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in ci-queue.gemspec
 gemspec
-
+gem 'byebug'
```

Comment:
Accidental? If not, perhaps we introduce a development scope?

Reply:
I just needed it to debug something, but yeah no big reason to include it in this PR.